src/cli/ui/message.go

```diff
@@ -38,4 +38,6 @@ const (
 
 	AggregatedMetricsNotFound = "No aggregated %s metrics were found for app %s."
 	HistoryNotFound           = "No event history were found for app %s."
+
+	MoreRecordsWarning = "There are more records, and only first page were displayed currently."
```

Comment:
Suggest to change the msg to "More records available. Please re-run the command with --start or --end option to fetch more"